lib/tasks/election.rake

```diff
@@ -59,4 +59,25 @@ namespace :election do
       puts e.message
     end
   end
+
+  desc "
+  Mark all documents of a given organisation as political
+  Usage:
+    rake election:mark_documents_as_political_for[organisation_slug]
+  "
+  task :mark_documents_as_political_for, [:slug] => :environment do |_t, args|
+    org = Organisation.find_by!(slug: args[:slug])
+    puts "Marking all documents as political for #{org.name}..."
+
+    editions = org.editions
+    editions.published.update_all(political: true)
+    editions.in_pre_publication_state.update_all(political: true)
+
+    org.editions.published.map(&:document_id).each do |document_id|
+      PublishingApiDocumentRepublishingWorker.perform_async_in_queue("bulk_republishing", document_id, true)
```

Comment:
Thinking about it @tw-raulgracia, we don't actually need to do this here, because when the election actually happens there's a task which republishes everything anyway: https://github.com/alphagov/whitehall/blob/00ae5b36e7ab8f219eaac3a1eaa8063030151351/lib/tasks/election.rake#L16

So we can drop the call to the republishing worker